torch/_dynamo/variables/tensor.py

```diff
@@ -707,15 +710,25 @@ def var_getattr(self, tx, name):
             result = wrap_fx_proxy_cls(
                 target_cls=NumpyNdarrayVariable,
                 tx=tx,
-                proxy=GetAttrVariable.create_getattr_proxy(self.as_proxy(), name),
+                proxy=tx.output.create_proxy(
+                    "call_function",
```

Comment:
This doesn't look right, if I'm accessing foo.T, this shouldn't show up as a function call in the resulting Dynamo graph. What exactly are you producing here?

Reply:
So the problem with foo.T is that because we delegate to torch_np, it's returning a torch_np.ndarray. We need to convert it back to Tensor.

Reply:
Also foo is actually a torch Tensor. So basically foo.T becomes `torch_np.ndarray(foo).T.tensor`